raven_aiohttp.py

```diff
@@ -19,30 +19,38 @@ class AioHttpTransport(AsyncTransport, HTTPTransport):
     def __init__(self, parsed_url, *, verify_ssl=True, resolve=True,
                  timeout=defaults.TIMEOUT,
                  keepalive=True, family=socket.AF_INET, loop=None):
+        self.verify_ssl = verify_ssl
+        self.resolve = resolve
```

Comment:
Should these instance variables names start with underscore?
They are cannot be changed after transport creation and they are not part of public API contract for transport.